### DISCUSSION TITLE
About the consistency of voices

### DISCUSSION THREAD

**Discussion by xdfnet** [lenML/Speech-AI-Forge#164] [问答 | Q&A]
How to keep the sound consistent before and after？

**Comment by zhzLuke96** [comment]
You can use `reference audio` to utilize the model's `ZeroShot(FewShot)` capability to maintain vocal consistency.

Additionally, the `ChatTTS` model can create random speakers using `seed` values while keeping the speaker's tone fixed.

I recommend using `reference audio` for more stability.
